stock_picking_comment_template_ext/models/sale_order.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    @api.multi
+    def _prepare_procurement_values(self, group_id=False):
+        values = super(SaleOrderLine, self)._prepare_procurement_values(group_id)
+        self.ensure_one()
+        values.update({"note2": group_id.sale_id.note2})
+        return values
```

Comment:
I guess we don't need this any more.

Reply:
OK I see. I will remove after check the behavior.